Dockerfile

```diff
@@ -2,31 +2,49 @@
 FROM node:22 AS frontend-build
 WORKDIR /frontend
 
-COPY app/frontend/ /frontend/
+COPY frontend/ /frontend/
```

Comment:
please revert the changes to this file, the existing dockerfile should still work as is. once the new one in the other repo exists we can fully delete it.